soc/xtensa/intel_adsp/common/include/intel_adsp_hda.h

```diff
@@ -383,4 +383,87 @@ static inline void intel_adsp_hda_underrun_clear(uint32_t base, uint32_t regbloc
 	*DGCS(base, regblock_size, sid) |= DGCS_BUR;
 }
 
+/**
+ * @brief Set the buffer segment ptr
+ *
+ * @param base Base address of the IP register block
+ * @param regblock_size Register block size
+ * @param sid Stream ID
+ * @param size size
```

Comment:
I know it's nitpicking but `@param size size` just look lazy.

Reply:
Good comments @tmleman, best to deal with the comments in a follow-up patch, it's a must have patch we're missing here and delays force the Linux kernel to carry a work-around